### DISCUSSION TITLE
Fundamentally broken parts of Golang SDK are still metioned in the official docs

### DISCUSSION THREAD

**Discussion by ipg0** [ory/.github#40] [Q&A]
The function `AdminApi.AcceptLoginRequest` is fundamentally broken and unusable: it sends requests with the field `subject` missing and there is no way (at least one that is documented or mentioned anywhere on the internet) to pass a subject to this API call. Hydra never accepts a call with a missing subject, the [AdminApi doc link](https://github.com/ory/hydra-client-go/blob/master/docs/AdminApi.md) on the [official website](https://www.ory.sh/docs/hydra/sdk/go) leads to a 404 page so it seems to have been deleted. But the same Go SDK doc I just mentioned still describes the process of using AdminApi. This is very confusing and leads to very significant waste of time. Is there any information on how to use this or why is it not removed at least from the front page of the doc?

**Comment by aeneasr** [comment]
Sorry to hear you have trouble. We use the SDK in several places with Go without issues. Can you please provide a reproducible example? Also please note that the SDKs have been updated with Ory Hydra 1.x :)
